### PR TITLE
Add support for indicate

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

File: src/gateway/hal/master_controller_core.py [hunk 1]
```diff
@@ -939,8 +939,25 @@ def _handle_firmware_information(self, information):  # type: (Dict[str, str]) -
     def replace_module(self, old_address, new_address):  # type: (str, str) -> None
         raise NotImplementedError('Module replacement not supported')
 
-    def flash_leds(self, led_type, led_id):
-        raise NotImplementedError()
+    def flash_leds(self, led_type, led_id):  # type: (int, int) -> str
+        """
+        Flash the leds on the module for an output/input/sensor.
+        :param led_type: The module type: output/dimmer (0), input (1), sensor/temperatur (2).
```
Comment (svanscho) [inline src/gateway/hal/master_controller_core.py:945]: typo

File: src/gateway/hal/master_controller_core.py [hunk 2]
```diff
@@ -939,8 +939,25 @@ def _handle_firmware_information(self, information):  # type: (Dict[str, str]) -
...
+        :param led_id: The id of the output/input/sensor.
+        """
+        if led_type not in [0, 1, 2]:
```
Comment (svanscho) [inline src/gateway/hal/master_controller_core.py:948]: move to constants

File: src/gateway/hal/master_controller_core.py [hunk 3]
```diff
@@ -939,8 +939,25 @@ def _handle_firmware_information(self, information):  # type: (Dict[str, str]) -
...
+        if led_type == 0:
+            output = OutputConfiguration(led_id)
+            if output.is_shutter:
+                self._master_communicator.do_basic_action(BasicAction(action_type=10, action=200, device_nr=led_id // 2))
+            else:
+                self._master_communicator.do_basic_action(BasicAction(action_type=0, action=200, device_nr=led_id))
+        elif led_type == 1:
+            self._master_communicator.do_basic_action(BasicAction(action_type=1, action=200, device_nr=led_id))
+        elif led_type == 2:
+            self._master_communicator.do_basic_action(BasicAction(action_type=8, action=200, device_nr=led_id))
```
Comment (svanscho) [inline src/gateway/hal/master_controller_core.py:959]: can't we use the core responses as we do for the classic master?

**Review by svanscho — CHANGES_REQUESTED** [review]

File: src/gateway/hal/master_controller_core.py (hunk 3 above)
Comment (khenderick) [reply src/gateway/hal/master_controller_core.py:959]: No, the Core returns other data (basically an echo of what was requested).

Comment (khenderick) [reply src/gateway/hal/master_controller_core.py:959]: No, the Core returns other data (basically an echo of what was requested).

**Review by svanscho — APPROVED** [review]

**Pull request merged**